src/helpers/date-helpers.ts

```diff
@@ -45,3 +45,18 @@ export function getMonthName(month: number) {
 	}
 	return capitalizeFirstLetter(format(new Date(new Date().getFullYear(), month, 1, 0, 0, 0, 0), "MMMM", { locale: locale }));
 }
+
+export function titleForDateRange(intervalType: "Day" | "Week" | "Month", intervalStart: Date, variant?: "short" | "long") {
+	if (intervalType == "Month" && intervalStart.getDate() == 1) {
+		return `${getMonthName(intervalStart.getMonth())} ${intervalStart.getFullYear()}`;
+	}
+	else if (intervalType == "Week" || (intervalType == "Month" && intervalStart.getDate() != 1)) {
+		return `${format(intervalStart, "MM/dd/yyyy")} - ${format(add(intervalStart, { days: -11 }), "MM/dd/yyyy")}`;
```

Comment:
Where does the hard coded `-11` days come from?

Reply:
...that's weird, must be a typo or something I messed up in moving this code around, definitely wrong

Reply:
Great catch on this, fixed...should have been subtracting one day from the interval end, somehow that got transformed into adding -11 days to interval start